R/shiny.R

```diff
@@ -478,6 +478,44 @@ ShinySession <- R6Class(
           # "json" unless requested otherwise. The only other valid value is
           # "rds".
           format <- params$format %||% "json"
+          # Machines can test their snapshot under different locales.
+          # R CMD check runs under the `C` locale.
+          # However, before this parameter, existing snapshots were most likely not
+          #   under the `C` locale is would cause failures. This parameter allows
+          #   users to opt-in to the `C` locale.
+          # From ?sort:
+          #   However, there are some caveats with the radix sort:
+          #     If ‘x’ is a ‘character’ vector, all elements must share the
+          #   same encoding. Only UTF-8 (including ASCII) and Latin-1
+          #   encodings are supported. Collation always follows the "C"
+          #   locale.
+          sortMethod <-
+            if (!is.null(params$sortC)) {
+              if (params$sortC != "1") {
+                stop("The `sortC` parameter can only be `1` or not supplied")
+              }
+              "radix"
+            } else {
+              # No `sortC` param
+              if (!file.exists("DESCRIPTION")) {
+                "auto"
+              } else {
+                # read from description file?
+                tryCatch({
+                  info <- as.list(read.dcf("DESCRIPTION")[1, , drop = TRUE])
```

Comment:
Convert to a `shinyOption`. Make sure to document that it's relevant for shinytest, but not shinytest2.